front/src/applications/editor/components/LayersModal.tsx

```diff
@@ -19,7 +19,7 @@ import OPsSVGFile from 'assets/pictures/layersicons/ops.svg';
 import switchesIcon from 'assets/pictures/layersicons/switches.svg';
 import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
 import { Modal } from 'common/BootstrapSNCF/ModalSNCF';
-import SwitchSNCF from 'common/BootstrapSNCF/SwitchSNCF/SwitchSNCF';
+import { SwitchSNCF } from 'common/BootstrapSNCF/SwitchSNCF';
```

Comment:
You can default export in SwitchSNCF/index and then:
```suggestion
import SwitchSNCF from 'common/BootstrapSNCF/SwitchSNCF';
```

Reply:
fixed !